pkg/kubelet/apis/well_known_labels.go

```diff
@@ -24,8 +24,10 @@ const (
 
 	LabelInstanceType = "beta.kubernetes.io/instance-type"
 
-	LabelOS   = "beta.kubernetes.io/os"
-	LabelArch = "beta.kubernetes.io/arch"
+	LabelArch      = "beta.kubernetes.io/arch"
+	LabelID        = "beta.kubernetes.io/id"
```

Comment:
nit: should this be something like `os-release-id`? `id` seems like it could take on a lot of values.